File: Modul-4/soal-4/PRAK404_2410817210017_OrlandoSugian.c
#include<stdio.h>
int main()
{
    int program;
    float angka1, angka2, hitung;
    while(1) {
        printf("Pilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan Pilihan : ");
        scanf("%d", &program);
        if (program == 5) {
            printf("Terimakasih, telah menggunakan kalkulator ORLANDOSUGIAN");
            break;
        }
        if (program < 1 || program > 4) {
            printf("Input anda salah, silahkan coba lagi\n\n");
            continue;
        }
        printf("Masukkan nilai pertama: ");
        scanf("%f", &angka1);
        printf("Masukkan nilai kedua: ");
        scanf("%f", &angka2);
        switch (program) {
            case 1:
                hitung = angka1 + angka2;
                printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n\n", angka1, angka2, hitung);
                break;
            case 2:
                hitung = angka1 - angka2;
                printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n\n", angka1, angka2, hitung);
                break;
            case 3:
                hitung = angka1 * angka2;
                printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n\n", angka1, angka2, hitung);
                break;
            case 4:
                hitung = angka1 / angka2;
                printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n\n", angka1, angka2, hitung);
    }
} return 0;
}

File: Modul-4/soal-4/PRAK404_2410817210017_OrlandoSugian.py
while True:
        print("Pilih program")
        print("1. Penjumlahan")
        print("2. Pengurangan")
        print("3. Perkalian")
        print("4. Pembagian")
        print("5. Exit")
        try:
            program = int(input("Masukkan Pilihan: "))
        except ValueError:
            print("Input anda salah, silahkan coba lagi\n")
            continue

        if program == 5:
            print("Terimakasih, telah menggunakan kalkulator ORLANDOSUGIAN")
            break

        if program not in [1, 2, 3, 4]:
            print("Input anda salah, silahkan coba lagi\n")
            continue
        try:
            angka1 = float(input("Masukkan nilai pertama: "))
            angka2 = float(input("Masukkan nilai kedua: "))
        except ValueError:
            print("Input anda salah, silahkan coba lagi\n")
            continue
        if program == 1:
            hasil = angka1 + angka2
            hitungan = "Penjumlahan"
        elif program == 2:
            hasil = angka1 - angka2
            hitungan = "Pengurangan"
        elif program == 3:
            hasil = angka1 * angka2
            hitungan = "Perkalian"
        elif program == 4:
            hasil = angka1 / angka2
            hitungan = "Pembagian"
        print(f"Hasil {hitungan} antara {angka1:.2f} dengan {angka2:.2f} adalah {hasil:.2f}\n")
